openbook_common/tests/helpers.py

```diff
@@ -37,6 +37,12 @@ def make_user(username=None):
     return user
 
 
+def make_user_with_invite_count(invite_count=5):
```

Comment:
just add it as an optional parameter to the `make_user`

Reply:
👍